app/models/service_update.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class ServiceUpdate
+  CLAIMS_SERVICE_UPDATES_YAML_FILE =
+    Rails.root.join("db/claims_service_updates.yml").freeze
+  PLACEMENTS_SERVICE_UPDATES_YAML_FILE =
+    Rails.root.join("db/placements_service_updates.yml").freeze
+
+  include ActiveModel::Model
+
+  attr_accessor :date, :title, :content
+
+  def id
+    title.parameterize
+  end
+
+  def self.where(service:)
```

Comment:
In rails where is used for return value that would be a list. I think what you want is a get/find